usersimcrs/core/information_need.py

```diff
@@ -0,0 +1,81 @@
+"""Interface to represent an information need.
+
+The information need is divided into two parts: constraints and requests. The
+constraints specify the slot-value pairs that the item of interest must satisfy,
+while the requests specify the slots for which the user wants information.
+"""
+
+from __future__ import annotations
+
+import random
+from collections import defaultdict
+from typing import Any, Dict, List
+
+from usersimcrs.core.simulation_domain import SimulationDomain
+from usersimcrs.items.item_collection import ItemCollection
+
+random.seed(42)
```

Comment:
Perhaps this should be controlled by some global `fix_random_seed: bool` flag?
This is a bit 'hidden' here and could cause some headache.

Reply:
Done, the random seed is now fixed for the entire program in `run_simulation.py` in case the configuration spitulates it.